civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1025,6 +1025,50 @@ function civihr_employee_portal_views_pre_render(&$view) {
   if ($view->name == "approvals") {
     $view->result = removeAbsencesForOtherManagers($view->result);
   }
+  
+  if ($view->name == "civihr_report_people") {
+    formatLengthsOfService($view->result);
+  }
+}
+
+/**
+ * Processes results in people report view to format length of service for each
+ * contact in a human readable form.
+ * 
+ * @param array $results
+ *   Array of objects (stdClass) that hold data for each contact in people report.
+ */
+function formatLengthsOfService(&$results) {
```

Comment:
since the contents of `$results` are objects, you don't need to pass by reference here (objects are always references):
https://3v4l.org/7DNTN